appinventor/appengine/src/com/google/appinventor/client/editor/simple/SimpleVisibleComponentsPanel.java

```diff
@@ -35,7 +36,7 @@
 public final class SimpleVisibleComponentsPanel extends Composite implements DropTarget, ComponentDatabaseChangeListener {
   // UI elements
   private final VerticalPanel phoneScreen;
-  private final CheckBox checkboxShowHiddenComponents;
+  public final CheckBox checkboxShowHiddenComponents;
```

Comment:
Rather than making this public, add a getter to retrieve the information you need.